src/js/view/utils/views-manager.js

```diff
@@ -45,6 +48,24 @@ function scheduleResponsiveRedraw() {
     });
 }
 
+function onOrientationChange() {
+    views.forEach(view => {
+        if (view.model.get('visibility') >= 0.75) {
+            const state = view.model.get('state');
+            const orientation = window.screen.orientation.type;
+            const isLandscape = orientation === 'landscape-primary' || orientation === 'landscape-secondary';
+
+            if (!isLandscape && state === 'paused') {
+                // Set fullscreen to false when going back to portrait while paused and return early
+                view.api.setFullscreen(false);
```

Comment:
Shouldn't you check if this api is in fullscreen mode first? What if it isn't but another visible one is?